samples/msgext-ai-sentiment-analysis/nodejs/Readme.md

```diff
@@ -0,0 +1,185 @@
+---
+page_type: sample
+description: This sample shows a feature where user can check sentiment for teams' chat messages by using messaging extension and Azure Open AI.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "08/07/2023 04:00:00 PM"
+urlFragment: officedev-microsoft-teams-samples-msgext-ai-sentiment-analysis-nodejs
+---
+
+# Sentiment Analysis for Teams chat messages using Azure Open AI and messaging extension.
+
+This sample shows a feature where user can analyze sentiments for messages posted in Teams chat using messaging extension and Azure Open AI.
+
+## Included Features
+* ME
+* Azure Open AI For Sentiment Analysis
+
+## Interaction with app
+
+![Sentiment Analysis](Images/Sentiment_Analysis.gif)
```

Comment:
Have you updated gif to remove bot card?

Reply:
gif updated

Reply:
@Mohammed-MSFT - Please check gif again.